src/Core/Testing/Functions.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Core\Testing;
+
+/**
+ * Class Functions is used to hold static functions required for testing
+ *
+ * @experimental
+ * @internal
+ */
+class Functions
```

Comment:
My goal was to try to use psr-4 to simplify the composer files, so that we don't have to have a "files" entry. Because we want these functions to be available from the google/cloud-core package as well as google/cloud, it means both composer files would need to have a "files" section.

Reply:
Keeping the name `Functions` seems a bit odd to me. Could we use something else? `TestHelpers`, maybe?